gigascience/gigadb-website
# Admin styles - Dataset view page

# Pull request for issue: #1512

This is a pull request for the following functionalities:

* Update styles for all admin pages that look similar to http://gigadb.gigasciencejournal.com:9170/datasetLog/view/id/4790

## How to test?

Navigate to all similar pages and cross check that styles have been updated

Here's the full list:

- http://gigadb.gigasciencejournal.com:9170/adminDatasetAuthor/view/id/253
- http://gigadb.gigasciencejournal.com:9170/adminDatasetSample/view/id/151
- http://gigadb.gigasciencejournal.com:9170/adminDatasetSample/view/id/151
- http://gigadb.gigasciencejournal.com:9170/adminFile/view/id/446
- http://gigadb.gigasciencejournal.com:9170/adminDatasetProject/view/id/124
- http://gigadb.gigasciencejournal.com:9170/adminLink/view/id/83
- http://gigadb.gigasciencejournal.com:9170/adminRelation/view/id/247
- http://gigadb.gigasciencejournal.com:9170/datasetFunder/view/id/1
- http://gigadb.gigasciencejournal.com:9170/adminManuscript/view/id/41
- http://gigadb.gigasciencejournal.com:9170/adminAuthor/view/id/14
- http://gigadb.gigasciencejournal.com:9170/adminSample/view/id/151
- http://gigadb.gigasciencejournal.com:9170/adminSpecies/view/id/5
- http://gigadb.gigasciencejournal.com:9170/adminProject/view/id/2
- http://gigadb.gigasciencejournal.com:9170/adminExternalLink/view/id/3
- http://gigadb.gigasciencejournal.com:9170/adminLinkPrefix/view/id/1
- http://gigadb.gigasciencejournal.com:9170/funder/view/id/1
- http://gigadb.gigasciencejournal.com:9170/attribute/view/id/1
- http://gigadb.gigasciencejournal.com:9170/adminDatasetType/view/id/2
- http://gigadb.gigasciencejournal.com:9170/adminFileType/view/id/1
- http://gigadb.gigasciencejournal.com:9170/adminFileFormat/view/id/1
- http://gigadb.gigasciencejournal.com:9170/user/view/id/8
- http://gigadb.gigasciencejournal.com:9170/rssMessage/view/id/1
- http://gigadb.gigasciencejournal.com:9170/adminPublisher/view/id/1
- http://gigadb.gigasciencejournal.com:9170/datasetLog/view/id/4790
- http://gigadb.gigasciencejournal.com:9170/news/view/id/2

## How have functionalities been implemented?

Implementation follows these steps:

- Use new layout in controllers

```
$this->layout = 'new_datasetpage';
```
- wrap page in `<div class="container">`
- Add breadcrumb
```
$this->widget('TitleBreadcrumb', [
  'pageTitle' => 'View XXX #' . $model->id,
  'breadcrumbItems' => [
    ['label' => 'Admin', 'href' => '/site/admin'],
    ['label' => 'Manage', 'href' => '/XXX/admin'],
    ['isActive' => true, 'label' => 'View'],
  ]
]);
```
- Delete unused variables (old breadcrumb and menu variables)
- Add this to the CDetailView widget
```php
'htmlOptions' => array('class' => 'table table-striped table-bordered dataset-view-table'),
'itemCssClass' => array('odd', 'even'),
'itemTemplate' => '<tr class="{class}"><th scope="row">{label}</th><td>{value}</td></tr>'
```

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
